doc/contributing/technical-priorities.md

```diff
@@ -160,3 +160,17 @@ and integration with other systems.
 As TypeScript usage continues to grow and gains more prominence in the
 ecosystem, enhancing its support is essential for delivering an improved
 developer experience for newcomers and experienced users alike.
+
+## Package management
+
+The ability to easily install and manage dependencies and development tools is a
+key part of the user experience, and for that reason Node.js must provide a
+package manager as part of its distribution. Node.js includes `npm` for this
+purpose. This is for historical reasons — when `npm` was added in 2011, it was
+the only JavaScript package manager — and because it is the reference
+implementation for the npm registry, which is the de facto primary source for
+most JavaScript software. In accordance with our [policy][distribution-policy]
+of not including multiple dependencies or tools that serve the same purpose, the
+Node.js project does not include any other package managers.
+
+[distribution-policy]: ./distribution.md
```

Comment:
I guess we could simply remove that sentence if folks find it confusing.
```suggestion
most JavaScript software.
```

Reply:
No, this is one of the key parts of the PR, stating that we don’t plan to include other package managers. We could change “include” to “vendor” perhaps, or add language like “the Node.js project does not include any other package managers, though it may include software to download other package managers.”

Reply:
Sure but it's already covered by the distribution policy, it's not something this PR is adding. If you think it's important for clarification purposes, let's go all-in on clarification
```suggestion
most JavaScript software. In accordance with our [policy][distribution-policy]
of not including multiple dependencies or tools that serve the same purpose, the
Node.js distribution does not include any other package managers, and include
Corepack that can let users use other package managers.

[distribution-policy]: ./distribution.md
```

Reply:
But npm can also install other package managers. We don’t need to mention Corepack specifically here, and that’s outside the scope of this PR.

```suggestion
most JavaScript software. In accordance with our [policy][distribution-policy]
of not including multiple dependencies or tools that serve the same purpose, the
Node.js project does not include any other package managers; though it may
include software to download other package managers.

[distribution-policy]: ./distribution.md
```

Reply:
I don't understand why mentioning npm is important, but not Corepack tbh
```suggestion
most JavaScript software. In accordance with our [policy][distribution-policy]
of not including multiple dependencies or tools that serve the same purpose, the
Node.js project does not include any other package managers; though it may
include other softwares to download other package managers.

[distribution-policy]: ./distribution.md
```

Reply:
Because the point of this PR is npm 😄 

```suggestion
most JavaScript software. In accordance with our [policy][distribution-policy]
of not including multiple dependencies or tools that serve the same purpose, the
Node.js project does not include any other package managers; though it may
include other software to download other package managers.

[distribution-policy]: ./distribution.md
```

Reply:
> this PR aims to help clarify the goals of Corepack